source-map.bs

```diff
@@ -302,7 +302,7 @@ fields [=offset=] and [=map=].
 ## Section
 
 <dfn><code>offset</dfn></code> is an object with two fields, `line` and `column`,
-that represent the offset into generated code that the referenced source map
+that represents the offset into generated code that the referenced source map
```

Comment:
"two fields [line and column] that represent the offset into ..." - I think both version work but I actually believe the existing choice ("that" refers to the more recent noun) sounded clearer.